tests/__main__.py

```diff
@@ -10,8 +12,14 @@
 os.putenv("DVC_HOME", REPO_ROOT)
 os.putenv("DVC_TEST", "true")
 
+if len(sys.argv) == 1:
+    scope = "--all-modules"
+else:
+    scope = " ".join(sys.argv[1:])
+
 cmd = (
     "nosetests -v --processes=-1 --process-timeout=500 --cover-inclusive "
-    "--cover-erase --cover-package=dvc --with-coverage --all-modules --with-flaky"
+    "--cover-erase --cover-package=dvc --with-coverage --with-flaky "
+    "{scope} ".format(scope=scope)
```

Comment:
:zap: This was the only reason I was using `pytest` instead of `python -m tests`!

Also, can we change the name to `extra-args` or something like that? I'm thinking of using `python -m tests -x` or `python -m tests -s tests/test_something.py`

Reply:
I'm not I understand your suggestion, could you give a specific example why would we need `-x` or `-s`?

Reply:
```
-x, --stop            Stop running tests after the first error or failure
-s, --nocapture       Don't capture stdout (any stdout output will be printed immediately) 
```

`-s` to make it work with `import ipdb; ipdb.set_trace()` (sometimes I need a step by step debugger to understand what's going on, I find it more broad to explore the codebase than `print`ing some lines)

Reply:
probably `scope` is the wrong var name, I think it should work if you pass any arguments, (you will have to specify the scope though) - `python -m tests -s -x tests` - is it good enough, @mroutis ? what do you think?

Reply:
Btw, if you use debugger I would strongly recommend using PyCharm - you can run any test with one click and have really nice debugger with a lot nice UI features.

Reply:
Thanks for the recommendation, I'll definetly take a look into it, maybe is time to bury the _good ol' Vim_.

`python -m tests -s -x tests` works for me :)

Would you mind editing the README.md to add information about how to run a single test?

Reply:
Totally, will update it!

Reply:
Will be updating the contributing guide, not README.md